Create JS Samples for Azure On Your Data (#85)
Azure-Samples/azureai-samples: scenarios/bring-your-own-data/README.md

---
page_type: sample
languages:
- JS
products:
- ai-services

> - openai-node
> description: Use AzureOpenAI client in the openai-node library and connect with on your data 

Reviewer: ```suggestion
- openai
description: Chat on top of and analyze your data using Azure OpenAI On Your Data which runs advanced AI models such as GPT-35-Turbo and GPT-4 on your own enterprise data without needing to train or fine-tune models.
```

---

> ## Use Your Data with AzureOpenAI client 

Reviewer: ```suggestion
## Use the AzureOpenAI client on your data
```

### Overview

> This tutorial provides a step-by-step guide on how to use AzureOpenAI client in `openai-node` library and generate chat completions on your data. It covers the entire workflow, from data preparation and model development to deployment and monitoring.

Reviewer: ```suggestion
This tutorial provides a step-by-step guide on how to use AzureOpenAI client in the [`openai`](https://www.npmjs.com/package/openai) library and generate chat completions on your data.
```

### Objective

> The main objective of this tutorial is to help users understand the process of creating, training, and deploying an AI model in Azure. By the end of this tutorial, you should be able to:

Reviewer: ```suggestion
The main objective of this tutorial is to help users understand the process of chatting on top of your own data. By the end of this tutorial, you should be able to:
```

>  - Connect to Azure using AzureOpenAI client

Reviewer 2: ```suggestion
 - Connect to Azure using an Azure OpenAI client
```

 - Get chat response with your own data

> ### Programming Languages

Reviewer: ```suggestion
### Programming languages
```

>  - Javascript

Reviewer 2: ```suggestion
 - JavaScript
```

> ### Estimated Runtime: 30 mins

Reviewer: ```suggestion
### Estimated Runtime: 5 mins
```
